### PR TITLE
feat: add schemas validation

### PR DESCRIPTION
**Pull request by Arcle1**
Add validation of schemas:
- package.json
- deps.json

### CODE REVIEW

File: script/gulp/config.json [hunk 1]
```diff
@@ -4,5 +4,9 @@
     },
     "build": {
         "dir": "build/"
+    },
+    "schemas": {
+        "package": "build/{**, **/**}/package.json",
```
Comment (svoboda-rabstvo) [inline script/gulp/config.json]: I think better mask is `./build/**/package.json`

Comment (Arcle1) [reply script/gulp/config.json]: @svoboda-rabstvo is mask for schemas in folders with version name (f.e. `alex/6.0.0`) shouldn't be `./build/**/**/package.json`?

File: script/validate.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const schema = require('@linterhub/schema');
+const jsonSchemaValidator = require('jsonschema').Validator;
+const validator = new jsonSchemaValidator();
+
+// Run validate task
+const validate = (mask) => {
+    glob.sync(mask).map((file) => {
+        const json = JSON.parse(fs.readFileSync(file));
+        const result = validator.validate(json, json.$schema);
+        if (result.errors.length > 0) console.log(result);
```
Comment (svoboda-rabstvo) [inline script/validate.js]: We need to print `throw` if exist any errors

File: script/validate.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    });
+};
+
+// Upload all validation schemas
+const upload = () => {
+    Object.keys(schema).map((schemaName) => {
+        validator.addSchema(schema[schemaName], schema[schemaName].$id);
```
Comment (svoboda-rabstvo) [inline script/validate.js]: I suggest to rename `schemaName` to `name` or `key`

File: script/validate.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+// Run validate task
+const validate = (mask) => {
+    glob.sync(mask).map((file) => {
+        const json = JSON.parse(fs.readFileSync(file));
+        const result = validator.validate(json, json.$schema);
+        if (result.errors.length > 0) console.log(result);
+    });
+};
+
+// Upload all validation schemas
```
Comment (svoboda-rabstvo) [inline script/validate.js]: Description don't say full information about function

**Pull request merged**